README.adoc

```diff
@@ -163,13 +163,24 @@ configuration management process.
 === Prerequisites
 
 * Install repo and docker if it is not already done.
+* Install make and pkg-config if t is not already done.
```

Comment:
I will merge the 2 lines and describe after how to install the prerequisites' packages.
For example:
```
 ==== Docker installation

See docker manual: https://docs.docker.com/engine/install/[Install docker]

==== Repo installation
```